index.html

```diff
@@ -53,6 +53,149 @@ <h1  class="headline1">I'm Loukmane
     </ul>
        </section>
 
+       <section id="workProjects">
+         <ul class="grid-container">
+           <li class="work work1">
+          <img class="img1" src="portfolioimgs/Snap1.png" alt="work1Snapshot">
+            <h1>Tonic</h1>
+            <div class="canopy">
+            <h3>CANOPY</h3>
+            <ul class="cards">
+              <li class="info">Back End Dev</li>
+              <li class="info">2015</li></ul>
+            </div>
+            <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
```

Comment:
- [ ] Please make sure that your font a colors match the Figma template as much as posible.
Yours
![image](https://user-images.githubusercontent.com/28109626/148471236-9ed82812-7213-431c-a375-75724db586b0.png)
Figma
![image](https://user-images.githubusercontent.com/28109626/148471262-46da902b-a0c4-4e05-a850-e182915ac64c.png)

Reply:
Please notify that the color used for the mentioned section match the color used in the Figma design